src/main/java/testproject/TestProject.java

```diff
@@ -1,11 +1,15 @@
 package testproject;
 
+import java.math.BigDecimal;
+
 public class TestProject {
 	
 	public static void main(String[] args) {
 		String alternativeName = "Max";
+		BigDecimal youShouldNotDoThis = new BigDecimal(1.5d);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "youShouldNotDoThis" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.rz.is/coding_rules#rule_key=squid%3AS1481)